scripts/airgap/image-list.txt

```diff
@@ -1,5 +1,5 @@
-docker.io/rancher/klipper-helm:v0.6.6-build20211022
-docker.io/rancher/klipper-lb:v0.3.4
+docker.io/rancher/klipper-helm:v0.6.8-build20211123
```

Comment:
You also need to bump the version of helm-controller in go.mod to a version that uses this as the job image.

Reply:
@brandond Can you review this ? https://github.com/k3s-io/helm-controller/pull/131

Reply:
Done in https://github.com/k3s-io/k3s/pull/5018/commits/2ec0bc4b841ad30c3a7e8a72a3595ee95e0f10ea